.github/workflows/cypresstests.yml

```
@@ -40,18 +40,25 @@ jobs:
       # This replaces the json file, do it before starting the server
       run: ./feed_dev_database.sh
 
+    - name: Bundle files
+      # Generates js and css files.
+      run: |
+        build-tools/heroku/tailwind/generate-development-css
+        build-tools/heroku/generate-typescript --tests
```

Can you add one commit where you remove this flag, to prove that this behavior is what is making the difference?

```suggestion
        build-tools/heroku/generate-typescript
```

---

Let's see what breaks if we commit this :)